manifest.lkml

```diff
@@ -3,3 +3,10 @@ project_name: "spoke-default"
 local_dependency: {
   project: "looker-hub"
 }
+
+# LAMS
+# rule: no_view_wildcards {
+#  description: "Please don't use wildcard view include (e.g. include: 'views/*') to avoid increasing LookML verification time. Explicitly list views to be included."
+#  match: "$.file.*..include.*"
+#  expr_rule:  ($not ($boolean ($match "views/.*\\*" ::match))) ;;
+# }
```

Comment:
IMO using a 2-space indent in this LookML-ish code like we do in most LookML would help with readability.
```suggestion
# rule: no_view_wildcards {
#   description: "Please don't use wildcard view include (e.g. include: 'views/*') to avoid increasing LookML verification time. Explicitly list views to be included."
#   match: "$.file.*..include.*"
#   expr_rule:  ($not ($boolean ($match "views/.*\\*" ::match))) ;;
# }
```